src/devoApp/DevoApp.Base.ts

```diff
@@ -22,9 +23,9 @@ import { AlertsConfig } from '../clients/alerts/AlertsConfig';
  * @internal
  */
 export abstract class DevoAppBase implements IDevoApp {
-  public async getQueryClient(): Promise<IQueryClient> {
+  public async getQueryClient(appInfo: AppInfo): Promise<IQueryClient> {
```

Comment:
Did we add a bit of documentation about this new parameter? I would add tsdoc here for the constructor. But as it's mandatory at a high level (to use the query client), we could add it in the readme. There's no example of the query client in the readme, though

Reply:
Done